src/LogStream.php

```
@@ -0,0 +1,362 @@
+     *
+     * @return array{status: string, time: string, ws_connections: int}|null
+     *         Returns null if the request failed or response is not JSON.
+     */
+    public function health(): ?array
+    {
+        $response = $this->request->get(self::ENDPOINT_HEALTH, [
+            "User-Agent: {$this->userAgent}",
+            'Accept: application/json',
+        ]);
+
+        if ($response->statusCode !== 200) {
```

PHPStan (2.1.44)

[error] Access to private property GuiBranco\Pancake\Response::$statusCode.


(property.private)